### PR TITLE
ci(deploy): fix the triggers

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: change/@vonage-vivid-43b600e8-568b-4e5d-bbd2-830928c2593e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Make a change",
```
Comment (yinonov) [inline change/@vonage-vivid-43b600e8-568b-4e5d-bbd2-830928c2593e.json:3]: oh did you?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: .github/workflows/_publish-package.yml [hunk 2]
```diff
@@ -19,11 +19,16 @@ jobs:
...
           git config --global user.name "Vivid CI"
-          npx beachball publish -m "Publishing a new version" -y
+
+      - name: Bump and Changelog
+        id: bump
+        run: |
+          npx beachball publish -m "Publishing a new version" --no-push -y > tmp.txt
+          echo "::set-output name=no_version_bump::$(if grep -q "Skipping publish" "./tmp.txt"; then echo true; else echo false; fi)"
 
       - name: Get us to the update version
```
[the remark this answers is not in the document]
Comment (YonatanKra) [reply .github/workflows/_publish-package.yml:33]: This takes `beachball`'s output of weather there was a bump or not and sets it in a boolean.

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  group: ci-pre-release-${{ github.ref }}-1
+  cancel-in-progress: true
+
+jobs:
+  call-deploy-site:
+    uses: vonage/vivid-3/.github/workflows/_deploy-deno.yml@main
+
+  remove-deploy-tag:
+    runs-on: ubuntu-latest
+    needs: [ call-deploy-site ]
```
[the remark this answers is not in the document]
Comment (YonatanKra) [reply .github/workflows/deploy.yml:20]: No - we have one that triggers on call and one that triggers on push the main (which uses the reusable workflow).

File: .github/workflows/_publish-package.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (yinonov) [reply .github/workflows/_publish-package.yml:33]: maybe rephrase?

**Review by yinonov — APPROVED** [review]
